root/etc/cont-init.d/98-dashboard-config

```diff
@@ -2,12 +2,17 @@
 
 echo "Applying the SWAG dashboard mod..."
 
-apk add --no-cache --upgrade goaccess
-
+apk add libmaxminddb
```

Comment:
this should have `--no-cache`, which is the rough equivalent of `apt update`, but then again, the package is already installed in swag